aws/rust-runtime/aws-config/src/sts/assume_role.rs

```diff
@@ -191,9 +191,11 @@ impl AssumeRoleProviderBuilder {
     /// Build a credentials provider for this role authorized by the given `provider`.
     pub fn build(self, provider: impl ProvideCredentials + 'static) -> AssumeRoleProvider {
         let conf = self.conf.unwrap_or_default();
-        let credentials_cache = CredentialsCache::lazy_builder()
-            .configure(conf.sleep(), conf.time_source())
-            .into_credentials_cache();
+
+        let mut builder = CredentialsCache::lazy_builder().time_source(conf.time_source());
```

Comment:
This cache should be configurable when creating an `AssumeRoleProvider`.

Reply:
👍  Addressed in b987538.